CultivateMVP/JournalOptionsScene/JournalOptionsViewModel.swift

```diff
@@ -1,17 +1,37 @@
 import Foundation
+import RxCocoa
+import RxSwift
 
 protocol JournalOptionsViewModelType {
-    var entries: [Entry] { get }
-    var moodEntry: MoodEntry { get }
+    var viewState: Driver<JournalOptionsViewState> { get }
 }
 
 struct JournalOptionsViewModel: JournalOptionsViewModelType {
 
-    var entries: [Entry]
-    var moodEntry: MoodEntry
+    var viewState: Driver<JournalOptionsViewState> = .never()
+    private var provider: JournalOptionsProvider
+
+    init(journalOptionsProvider: JournalOptionsProvider) {
```

Comment:
I think we can hardcode the questions. So you can just inject a hardcoded array. 
`init(journalQuestions: JournalQuestions = .default)`
-> 
```swift
strut JournalQuestions {
 let questions: [JournalQuestion]

static let default = <your harcoded stuff>
}
```